ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -128,6 +128,13 @@ <h3> Generated Resources:</h3>
             </li>
         </span>
     </div>
-    <button mat-raised-button type="submit" color="primary" (click)="migrate()" class="migrate"
-        [disabled]="!((isTargetDetailSet && (selectedMigrationType ==='lowdt') && isSourceConnectionProfileSet && isTargetConnectionProfileSet) || (isTargetDetailSet && (selectedMigrationType ==='bulk'))) || isMigrationInProgress">Migrate</button>
+    <div class="migrate">
+        <span *ngIf="!isLowDtMigrationComplete">
+            <button mat-raised-button type="submit" color="primary" (click)="migrate()"
+                [disabled]="!((isTargetDetailSet && (selectedMigrationType ==='lowdt') && isSourceConnectionProfileSet && isTargetConnectionProfileSet) || (isTargetDetailSet && (selectedMigrationType ==='bulk'))) || isMigrationInProgress">Migrate</button>
+        </span>
+        <span *ngIf="isLowDtMigrationComplete">
```

Comment:
nit: This flag should probably be names - isLowDtMigrationRunning ?

Reply:
done